concent_api/command_line_tool/message_handler.py

```diff
@@ -70,9 +70,10 @@ def _exchange_message(self, priv_key, cluster_url, data):
         elif response.status_code == 204:
             print('')
             print('STATUS: 204 No Content')
-        elif response.status_code in [400, 404, 500]:
+        elif response.status_code in [400, 404, 500, 503]:
```

Comment:
Please don't hardcode specific errors.

Treat any 4xx as a client error.
Treat any 5xx as a server error.
Treat any 3xx as something unexpected (should not happen in practice).